doc/api/esm.md

```diff
@@ -0,0 +1,85 @@
+# Modules
+
+<!--introduced_in=v9.x.x-->
```

Comment:
I think you can still use `REPLACEME` for the version here (which gets replaced as part of the standard release process)